src/Pim/Component/Catalog/Normalizer/Indexing/Product/PropertiesNormalizer.php

```diff
@@ -29,7 +29,10 @@ public function normalize($product, $format = null, array $context = [])
         }
 
         $data = [];
+
         $data[StandardPropertiesNormalizer::FIELD_IDENTIFIER] = $product->getIdentifier();
+        $data[StandardPropertiesNormalizer::FIELD_CREATED] = $this->serializer->normalize($product->getCreated(), $format);
```

Comment:
if you don't change the ProductIndexingIntegration test, I think I'll kill you :p